qa/rolling-upgrade/src/test/java/org/opensearch/upgrades/IndexingIT.java

```diff
@@ -95,7 +87,8 @@ private void printClusterNodes() throws IOException, ParseException, URISyntaxEx
     private void waitForSearchableDocs(String index, int shardCount) throws Exception {
         Map<Integer,String> primaryShardToNodeIDMap = new HashMap<>();
         Map<Integer,String> replicaShardToNodeIDMap = new HashMap<>();
-        logger.info("--> _cat/shards \n{}", EntityUtils.toString(client().performRequest(new Request("GET", "/_cat/shards?v")).getEntity()));
+        waitForClusterHealthWithNoShardMigration(index, "green");
```

Comment:
nit, can we use ClusterHealthStatus here over strings.

Reply:
Used `green` string over multiple places in the test. I will resolve this in follow up PR.